### DISCUSSION TITLE
解决吃饭选择困难的智能小助手

### DISCUSSION THREAD

**Discussion by OceanLiu99** [creatorsand/.github#79] [Ideas]
**一句话 IDEA：**

针对外出餐饮/外卖的智能饮食助手。与助手的对话中进行一系列选择（重口/清淡；碳水/轻食；人均；多久可达etc），助手提出餐馆/外卖的一条龙规划

**我的角色：**

整体构想、原型设计

**其他角色需求（不包含发起人）：**

会编程的朋友

**社群昵称：**

Ocean

**个人简介：**

研二学生，做过一些产品流程梳理和figma原型（app和网页都有）~编程方面会一点python但不多

**产品是给谁用的？**

目标用户群体是打工人和大学生。应用于不知道吃什么的时候，饮食助手给出选择建议和一条龙服务（导航/联系方式/点菜建议/……）

**其他产品有什么痛点，用户为什么要选择你的产品？**

暂未发现可以集成美团+地图+健康功能的智能软件。想做一个餐饮软件中最会规划路线的，地图软件中最懂饮食的。还可以增加好友圈社交互动功能，餐饮地图里显示好友的种草/排雷建议

**Comment by ShirleneLiu** [comment]
请看👉🏻 ![image](https://github.com/user-attachments/assets/2ff23095-5d9e-4fee-b12f-d561bb755276)

**Reply by OceanLiu99** [reply]
哇就是想象里的一部分功能！

**Comment by Cobb9527** [comment]
一看就是个吃货哈哈哈~
